src/explorer/weights/WeightSlider.js

```diff
@@ -2,30 +2,61 @@
 
 import React from "react";
 
-export const WEIGHT_SLIDER_MIN = -5;
-export const WEIGHT_SLIDER_MAX = 5;
+/**
+ * The Weight is the number that the WeightSlider actually provides to the
+ * instantiating component. The Weight varies in the range
+ * [0, 2**MAX_SLIDER_VALUE].
+ */
+export type Weight = number;
+
+/**
+ * The SliderPosition is the current position of the [range input] that the
```

Comment:
Is `SliderPosition` intended to be an integer-valued type only?

What about `Weight`?

Reply:
SliderPosition is intended to be integer; Weight may be any finite nonnegative number. Updated the docstrings accordingly.

Reply:
SliderPosition should be integer. Weight can be any nonnegative finite number. I've updated docs accordingly.